src/assimp/SceneConverter.cpp

```diff
@@ -598,6 +598,9 @@ void SceneConverter::convert(const aiMaterial* material, vsg::DescriptorConfigur
 
         vsg::PhongMaterial mat;
 
+        if (filename.find(L"Cube-Standard-Brown-Low.fbx") != std::wstring::npos)
```

Comment:
?  A bit of debugging seems to have slipped through...

Reply:
Yeah, I staged it properly, then had to stash and unstash, and forgot I'd got other changes when I staged things again.

I wish that conditional breakpoints involving strings weren't such a pain with C++ in Visual Studio - I wouldn't have needed this in the first place.

Reply:
Fixed.